src/responses/fileNotFound.ts

```diff
@@ -1,6 +1,6 @@
 import { CACHE_HEADERS } from '../constants/cache';
 
-export default (request: Request): Response => {
+export default (request: Pick<Request, 'method'>): Response => {
```

Comment:
Why Pick? In the end you are passing the whole object, no? Or if just method indeed, shouldn't it just be `method: Request['method']` (?)

Reply:
Preference mostly, but `method: Request['method']` works as well